pandas/_libs/groupby_mode_helper.pxi.in

```diff
@@ -0,0 +1,177 @@
+{{py:
```

Comment:
To @jbrockmendel point a lot of this can be simplified if using fused types instead of a template; we've started to move to that in most other places in Cython